tests/test_signalsapi.py

```diff
@@ -70,3 +71,18 @@ def test_ticker_universe(api):
     result = api.ticker_universe()
     assert "bloomberg_ticker" not in result
     assert len(result) == 3
+
+
+def test_download_current_dataset(api, tmpdir):
```

Comment:
this is downloading the file during the test, would be better to use a mock. (I know that the current codebase does the same)